src/pages/VoucherRedemption/Landing.tsx

```diff
@@ -95,9 +100,9 @@ const SupportingText = styled.span`
 `;
 const Button = styled.div`
   cursor: pointer;
-  ${(props: ButtonProps) => props.color && `color: ${props.color}`}
+  ${(props: ButtonProps) => props.color && `color: ${props.color};`}
```

Comment:
unrelated to your change, but does this line just add `undefined` when props.color is undefined? Maybe we should do a ternary operator with `''` if falsy

Reply:
Cool I got it to work using the null coalesce operator
`${(props: ButtonProps) => `color: ${props.color ?? ''};`}`